.github/workflows/cd.yml

```diff
@@ -0,0 +1,48 @@
+name: bosdyn_msgs CD
+
+on:
+  push:
+    tags:
+      - '*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-bundles-amd64:
+    runs-on:
+      group: bdai-gh-large-runners
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Build bosdyn_msgs bundles
+      run: make -j8 VERSION=${{ github.ref_name }} amd64
+
+    - name: Release bosdyn_msgs bundles
+      uses: softprops/action-gh-release@v1
+      with:
+        name: Release ${{ github.ref_name }} Amd64
+        files: '*.run'
+
+
+  release-bundles-arm64:
+    runs-on:
+      group: k8s-t2a-standard-16
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true 
+      
+      - name: Build bosdyn_msgs bundles
+        run: make -j8 VERSION=${{ github.ref_name }} arm64
+      
+      - name: Release bosdyn_msgs bundles
+        uses: softprops/action-gh-release@v1
+        with:
+          name: Release ${{ github.ref_name }} Arm64
+          files: '*.run'
```

Comment:
@amessing-bdai meta: I wonder how well this is going to work. It's a single release we are making. I would expect either release to fail or override the other. 

We could have one release per architecture. We could also [upload](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) binary as temporary artifacts and release once in a later, dependent job.

Reply:
I was thinking about that. It would be nice to do the single release